package.nls.json

```diff
@@ -101,6 +103,7 @@
     "azureFunctions.toggleAppSettingVisibility": "Toggle App Setting Visibility.",
     "azureFunctions.uninstallFuncCoreTools": "Uninstall Azure Functions Core Tools",
     "azureFunctions.validateFuncCoreTools": "Validate the Azure Functions Core Tools is installed before debugging.",
+    "azureFunctions.validateServiceConnector": "Validate Service Connector",
```

Comment:
Is this the right wording? Should it be "Validate connection" or "Validate"?

Reply:
Right now the portal just says Validate. I could also say `Validate connection status` since that is also shown in the portal 
![image](https://github.com/microsoft/vscode-azurefunctions/assets/59709511/92368f43-99d7-4cfd-8c5a-204bff7288c0)